docs/api-markdown-documenter/api-documentation-layout.js

```diff
@@ -18,25 +20,49 @@ import { AlertNode } from "./alert-node.js";
 const customExamplesSectionTitle = "Usage";
 const customThrowsSectionTitle = "Error Handling";
 
-const alphaWarning = SpanNode.createFromPlainText(
-	"WARNING: This API is provided as an alpha preview and may change without notice. Use at your own risk.",
-);
-const betaWarning = SpanNode.createFromPlainText(
-	"WARNING: This API is provided as a beta preview and may change without notice. Use at your own risk.",
-);
+// Temporary workaround for items tagged as `@alpha` (to mean "legacy").
+// This messaging should be changed back to standard "alpha" terminology once we have
+// cleaned up our tag meanings.
+function createAlphaWarning(apiItem) {
+	const packageName = apiItem.getAssociatedPackage().displayName;
+	return new AlertNode(
+		[
+			new SpanNode([
+				new PlainTextNode("To use, import via "),
+				CodeSpanNode.createFromPlainText(`${packageName}/legacy`),
+				new PlainTextNode("."),
+			]),
+		],
+		/* alertKind: */ "note",
+		/* title: */ "This API is provided as-is for existing users, but is not recommended for new users.",
```

Comment:
@kashms interested in your thoughts on this wording.

Reply:
Also wondering if we should note the support contract here. Presumably: will offer support for bugs, but are not taking feature requests around it.

Reply:
I think we should remove the `as-is` bit and keep the "This API is not recommended for new users" and link out to our FF.com documentation here for more information: https://fluidframework.com/docs/build/releases-and-apitags/#api-tags